Sources/Common/Core/Math/index.d.ts

```diff
@@ -771,14 +771,6 @@ export function uninitializeBounds(bounds: Bounds): Bounds;
  */
 export function areBoundsInitialized(bounds: Bounds): boolean;
 
-/**
- * Compute the bounds from points.
- * @param {Vector3} point1 The coordinate of the first point.
- * @param {Vector3} point2 The coordinate of the second point.
- * @param {Bounds} bounds Output array that hold bounds, optionally empty.
- */
-export function computeBoundsFromPoints(point1: Vector3, point2: Vector3, bounds: Bounds): Bounds;
```

Comment:
This is because this is a duplicate with vtkBoundingBox. 
Hopefully no-one use this function.
Ok to flag it as "deprecated" with an explanation.
All the `bounds` modifications should be done in a separate commit (same PR though)